railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -131,5 +131,18 @@
 # If your cookies can't yet be serialized to JSON, keep using `:marshal` for backward-compatibility.
 #
 # If you have configured the serializer elsewhere, you can remove this section of the file.
-# 
+#
 # See https://guides.rubyonrails.org/action_controller_overview.html#cookies for more information.
+
+# Change the return value of `ActionDispatch::Request#content_type` to the Content-Type header without modification.
+# Rails.application.config.action_dispatch.return_only_request_media_type_on_content_type = false
+
+# Active Storage `has_many_attached` relationships will default to replacing the current collection instead of appending to it.
+# Thus, to support submitting an empty collection, the `file_field` helper will render an hidden field `include_hidden` by default when `multiple_file_field_include_hidden` is set to `true`.
+# See https://guides.rubyonrails.org/configuring.html#config-active-storage-multiple-file-field-include-hidden for more information.
+# Rails.application.config.active_storage.multiple_file_field_include_hidden = true
+
+# ** Please read carefully, this must be configured in config/application.rb (NOT this file) **
+# Disables the deprecated #to_s override in some Ruby core classes
+# See https://guides.rubyonrails.org/configuring.html#config-active-support-disable-to-s-conversion for more information.
+# config.active_support.disable_to_s_conversion = true
```

Comment:
I'm not certain there's value for this one. It's typically the thing you'll get once you update `load_defaults`.

Reply:
I think it's important we have every option set by `load_defaults` in this file. Otherwise we end up with bug reports like #46118 and #46350 (the one this PR fixes). 

In addition, I think this configuration change in particular is important because it changes a reported deprecation to a raise. While good test coverage would theoretically prevent any deprecation from first appearing in production, I would rather let users knowingly opt into that behavior. (As a sidenote, this deprecation is also not mentioned in the upgrading guide, only the 7.0 release notes, making it less discoverable as well)

Reply:
+1 to what @skipkayhil said. It helps making the change to the config value for load defaults much less surprising and risky if we know what all is changing and make the changes individually first, as is the point of the new framework defaults file.